pele_platform/Checker/valid_flags.py

```diff
@@ -163,7 +163,7 @@
 "center_of_interface": "center_of_interface", 
 "protein": "protein", 
 "ligand_pdb": "ligand_pdb",
-"allosteric": "allosteric", 
+"site_finder": "site_finder",
```

Comment:
This is some sort of a API-breaking change. We need to highlight it in the release notes.

Reply:
Task to prepare release notes, with extra emphasis on this change. https://nostrumbiodiscovery.atlassian.net/browse/PELE-437